kanidm_unix_int/src/cache.rs

```diff
@@ -1,3 +1,4 @@
+use crate::constants::DEFAULT_SHELL;
```

Comment:
You shouldn't be importing this, rather you sohuld use the `self.default_shell` field on the instance. This way later we can have the default_shell be configurable, rather than hardcoded. This line should be removed, and all instances of DEFAULT_SHELL should be self.default_shell

See also, kanidm_unix_int/src/daemon.rs line 571 where we set the default shell based on the configuration file.

Reply:
alright, I will remove it. I was not sure about `self.default_shell`.